cmd/lakectl/cmd/log.go

```diff
@@ -44,14 +46,17 @@ var logCmd = &cobra.Command{
 		if err != nil {
 			DieErr(err)
 		}
+		showMetaRangeID, _ := cmd.Flags().GetBool("show-meta-range-id")
```

Comment:
Sure is better than `--format "%H"` (as the _only_ way I could find to convince git to show me a tree-ID)!